packages/apps/dev-wallet/src/pages/personalize-profile/personalize-profile.css.ts

```diff
@@ -0,0 +1,31 @@
+import { atoms, tokens } from '@kadena/react-ui/styles';
+import { style } from '@vanilla-extract/css';
+
+export const listClass = style([
+  atoms({
+    display: 'flex',
+    justifyContent: 'space-between',
+    listStyleType: 'none',
+    padding: 'n0',
+    width: '100%',
+    marginBlockEnd: 'xxxl',
+    marginBlockStart: 'sm',
+  }),
+]);
+
+export const colorOptionClass = style([
+  atoms({
+    borderRadius: 'xs',
+    cursor: 'pointer',
+  }),
+  {
+    width: '68px',
+    maxWidth: '100%',
+    aspectRatio: '1 / 1',
+    selectors: {
+      '&.active': {
+        border: `2px solid ${tokens.kda.foundation.color.palette.red.n50}`,
```

Comment:
as discussed at the refinement, it should be checked with UX (Julian) as red border can look like an validation error

Reply:
Julian has updated the design. I will implement the new one